web/json_options.go

```diff
@@ -75,6 +75,7 @@ func defaultJSONOptions() *jsonOptions {
 			{"01-02-2006", false},
 			{"01/02/2006", false},
 			{"01/02/06", false},
+			{"SGNLUnixSec", false}, // Unix timestamp representing seconds since 1970-01-01 00:00:00 UTC.
```

Comment:
Just a general question, does `time.Stamp` not work here? https://pkg.go.dev/time#pkg-constants.